machine/quanta/quanta_common_rangeley/busybox/patches/onie-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,722 @@
+Add support for onie-syseeprom and mb_detect
+
+diff --git a/include/applets.src.h b/include/applets.src.h
+index 1c8e0cb..a600632 100644
+--- a/include/applets.src.h
++++ b/include/applets.src.h
+@@ -417,6 +417,7 @@ IF_YES(APPLET_NOFORK(yes, yes, BB_DIR_USR_BIN, BB_SUID_DROP, yes))
+ IF_GUNZIP(APPLET_ODDNAME(zcat, gunzip, BB_DIR_BIN, BB_SUID_DROP, zcat))
+ IF_ZCIP(APPLET(zcip, BB_DIR_SBIN, BB_SUID_DROP))
+ IF_SYS_EEPROM(APPLET_ODDNAME(onie-syseeprom, onie_syseeprom, BB_DIR_USR_BIN, BB_SUID_DROP, onie_syseeprom))
++APPLET_ODDNAME(mb_detect, mbdetect, BB_DIR_USR_BIN, BB_SUID_DROP, mbdetect)
```

Comment:
CONFIG_MBDETECT=y will not applied to busybox .config because ``apply-config-patch`` script will filter this out.
I finally chose to remove the IF_XXXX prefix and make sure MBDETECT works.